extensions/data-plane/data-plane-aws-s3/src/main/java/org/eclipse/edc/connector/dataplane/aws/s3/S3DataSink.java

```diff
@@ -140,6 +141,10 @@ public Builder chunkSizeBytes(int chunkSize) {
         }
 
         @Override
-        protected void validate() {}
+        protected void validate() {
+            if (this.sink.chunkSize < 1) {
+                throw new IllegalArgumentException("Chunk size must be greater than zero! Actual value is: " + this.sink.chunkSize);
+            }
```

Comment:
please avoid this lazy validation, if the chunkSize is not configured correctly, the dataplane should not start at all, please validate it in the `initialize` method of the extension.

Reply:
I've added it into the **_initialize_** method of the extension and left it here as well.

Reply:
here does not make any sense anymore, please remove it